crypto-ffi/bindings/js/test/CoreCrypto.test.ts

```diff
@@ -133,6 +133,30 @@ test("init", async () => {
     await ctx.close();
 }, 10000);
 
+test("can get build data", async () => {
+    const [ctx, page] = await initBrowser();
+
+    const buildMetadata = await page.evaluate(async () => {
+        const { CoreCrypto, Ciphersuite } = await import("./corecrypto.js");
+
+        const ciphersuite =
+            Ciphersuite.MLS_128_DHKEMX25519_AES128GCM_SHA256_Ed25519;
+        await CoreCrypto.init({
+            databaseName: "test init",
+            key: "test",
+            ciphersuites: [ciphersuite],
+            clientId: "test",
+        });
```

Comment:
We don't need to initialize CoreCrypto, just to get build metadata, right?

Reply:
Strictly speaking we shouldn't. But I was basing things off the `init` test above, and the `version` implementation. The `version` implementation does `this.#assertModuleLoaded();`, so `buildMetadata` does also, so the test only passes if we init the instance first.

Reply:
So how does `setLogger` then work?
I think we should remove the `init`, it just doesn't make any sense here and needlessly complicates the test.

Reply:
Ok, https://github.com/wireapp/core-crypto/pull/767/commits/e2a36a304de70b49717aa8efdd674440498028d2 does that now.

Reply:
Undid that change. It turns out that we must in fact init CoreCrypto in order for this all to work. I don't know _why_ that's the case, but the tests fail if we try without it. So we're back to https://github.com/wireapp/core-crypto/pull/767/commits/dd249f7330007e272c3c6d3a1718e210ba9ae56b which both requires and performs an init before getting version or build info.

Reply:
I think it is important to know the cause of the test failure here. Please see my comment about `setLogger` above.

Reply:
I don't understand your point about `setLogger`. It is documented to always require `init` to be called first. All tests which call it, call an `init` first. 

https://github.com/wireapp/core-crypto/blob/6d9b073fd5a69982e23376733d87a1c591e391ab/crypto-ffi/bindings/js/CoreCrypto.ts#L920-L927

Reply:
Ugh, I was looking at an older version of the code, sorry.

In what way does the test fail if you don't call `init`?

Reply:
It was a relatively complicated error message and I don't recall the details, but in combination with the existing guard which fails the method if it is not initialized, I surmise initialization performs some essential binding between the TS `CoreCrypto` class and the actual WASM implementations.

Reply:
Hmm, that sounds plausible. It would be interesting to see whether we can have module-level functions working without having to init CC first. But oh well. One thing at a time.